.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,4 +13,5 @@ See: https://jira.bf.local/browse/XYZ
 - [ ] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
 - [ ] Public API has Javadoc
 - [ ] Method preconditions are checked and documented in the Javadoc of the method
+- [ ] Change is not breaking **or** mentioned in the Changelog
```

Comment:
We document some non-breaking changes too, if they significant to the user. Maybe,
> is not user-facing?

or

> mentioned … *if* notable/significant?

> Why keep a changelog?
To make it easier for users and contributors to see precisely what **notable** changes have been made between each release (or version) of the project.

Reply:
Good point